cli/cmd/root.go

```diff
@@ -60,6 +61,8 @@ func init() {
 	rootCmd.PersistentFlags().BoolVarP(&mocking, "mock", "", false, "Disables communication to a Keptn endpoint")
 	rootCmd.PersistentFlags().BoolVarP(&SuppressWSCommunication, "suppress-websocket", "", false,
 		"Disables WebSocket communication to suppress info messages from services running inside Keptn")
+	rootCmd.PersistentFlags().StringVarP(&namespace, "namespace", "n", "keptn",
```

Comment:
Does this mean we need to add the `-n` flag for every Keptn command? E.g., `keptn create project sockshop --shipyard=./shipyard.yaml -n keptn`? Or is this just temporary?

Reply:
That is when you install keptn in namespace other than `keptn`